cli/src/main.rs

```diff
@@ -203,6 +204,49 @@ fn run() -> Result<(), CliError> {
                                 .possible_values(&["accept", "reject"])
                                 .help("Reject the proposal"),
                         ),
+                )
+                .subcommand(
+                    SubCommand::with_name("list")
+                        .about("list the circuits")
```

Comment:
To match the other usage lines, these should start with a capital letter.

Reply:
updated